starky/src/lookup.rs

```diff
@@ -761,79 +761,35 @@ pub(crate) fn get_helper_cols<F: Field>(
         .len()
         .div_ceil(constraint_degree.checked_sub(1).unwrap_or(1));
 
-    let mut helper_columns = Vec::with_capacity(num_helper_columns);
```

Comment:
Iterators in Rust have a [size hint](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.size_hint), so `collect` automatically does what we were trying to do manually via `with_capacity` here.